lib/autoupdate/agent/config.go

```diff
@@ -59,8 +60,8 @@ type UpdateSpec struct {
 	Proxy string `yaml:"proxy"`
 	// Group specifies the update group identifier for the agent.
 	Group string `yaml:"group,omitempty"`
-	// URLTemplate for the Teleport tgz download URL.
-	URLTemplate string `yaml:"url_template,omitempty"`
+	// BaseURL is CDN base URL used in `url_template` for the Teleport tgz download URL.
```

Comment:
```suggestion
	// BaseURL is CDN base URL for the Teleport tgz download URL.
```
`url_template` is removed